webapp/src/components/DonationsDashboard/DonationsDashboard.js

```diff
@@ -0,0 +1,407 @@
+    zIndex: 1,
+    top: 10,
+    right: 10,
+    margin: '0',
+    height: "5vh",
+    '& svg': {
+      fontSize: 25,
+      color: "rgba(0, 0, 0, 0.6)"
+    }
+  },
+}));
+
+const EmptyHeartSVG = ({ balance, isDesktop }) => {
+  const classes = useStyles()
+  let textColor
+
+  if (isDesktop) {
+    textColor = "#ffffff"
+  } else {
+    textColor = "#000000"
+  }
+
+  return (
+    <svg viewBox="0 0 800 700" className={classes.heart}>
+      <path
+        fill="#B71C1C"
+        d="M514.672,106.17c-45.701,0-88.395,22.526-114.661,59.024c-26.284-36.505-68.981-59.024-114.683-59.024C207.405,106.17,144,169.564,144,247.5c0,94.381,57.64,144.885,124.387,203.358c38.983,34.149,83.17,72.856,119.654,125.332l12.267,17.641l11.854-17.924c35.312-53.388,78.523-91.695,120.305-128.734C596.006,390.855,656,337.654,656,247.5C656,169.564,592.604,106.17,514.672,106.17z M513.143,425.371c-36.93,32.729-78.27,69.373-113.402,117.391c-35.717-46.873-76.089-82.242-112.148-113.834c-63.944-56.01-114.447-100.26-114.447-181.428c0-61.868,50.325-112.186,112.184-112.186c43.196,0,83.034,25.395,101.491,64.697l13.191,28.105l13.19-28.112c18.443-39.303,58.273-64.69,101.472-64.69c61.866,0,112.185,50.317,112.185,112.186C626.856,324.548,576.673,369.047,513.143,425.371z"
+      />
+      {isDesktop &&
+        <g transform="translate(150, 200)">
+          <path
+            fill="#B71C1C"
+            d="M 10,30 A 25,22 0,0,1 250,10 A 25,20 0,0,1 480,10 Q 550,80 240,360 Q 10,150 10,30 z"
+          />
+        </g>
+      }
+      <text
+        x="50%"
+        y="50%"
+        dominantBaseline="middle"
+        textAnchor="middle"
+        fontSize="200"
+        fontFamily="Roboto"
+        textRendering="geometricPrecision"
+        lengthAdjust="spacingAndGlyphs"
+        fill={textColor}
+      >
+        {balance}
+      </text>
+    </svg>
+  )
+}
+
+EmptyHeartSVG.propTypes = {
+  balance: PropTypes.string,
+  isDesktop: PropTypes.bool,
+}
+
+const DonationsDashboard = ({ isDesktop }) => {
+  const [maxWidth] = useState('md');
+  const [open, setOpen] = useState(false)
+  const classes = useStyles();
+  const [state, setState] = useState({
+    bottom: false,
+  });
+  const [currentUser] = useUser()
+  const history = useHistory()
+  const [
+    loadProfile,
+    { data: { profile: { profile } = {} } = {}, client }
+  ] = useLazyQuery(PROFILE_QUERY, { fetchPolicy: 'network-only' })
+
+  const tokens = profile?.balance.length
+    ? profile.balance.join(',').split(' ')[0]
+    : 0
+
+  useEffect(() => {
+    if (!currentUser) {
+      client && client.resetStore()
+      history.replace('/')
+
+      return
+    }
+
+    loadProfile()
+  }, [currentUser, history, client, loadProfile])
+
+  const anchor = "bottom"
+
+  const toggleDrawer = (anchor, open) => (event) => {
+    if (event) {
+
+      if (event.type === 'keydown' && (event.key === 'Tab' || event.key === 'Shift')) {
+        return;
+      }
+
+      setState({ ...state, [anchor]: open });
+    }
+  };
```

Comment:
Check your format. We are using standard JS. It's not necessary to use semicolons